handlers/handler-publickey-auth/src/main/java/org/apache/servicecomb/authentication/provider/ProviderAuthFilter.java

```diff
@@ -50,11 +57,44 @@ public String getName() {
 
   @Override
   public CompletableFuture<Response> onFilter(Invocation invocation, FilterNode nextNode) {
+    if (checkPathWhitelist(invocation.getRequestEx().getServletPath())) {
```

Comment:
Do you mean request url by using `invocation.getRequestEx().getServletPath()` ?

Reply:
fixed